gui/WorldView.cxx

```diff
@@ -1035,13 +1087,40 @@ void WorldView::setMesh(vtkSmartPointer<vtkPolyData> mesh)
 
   // Add this actor to the renderer
   d->renderer->AddActor(d->volumeActor);
-
-  if (mesh->GetPointData()->HasArray("RGB"))
+  bool rgb = false, rgba = false;
+  const char* arrayName = nullptr;
+  const char* rgbName = "RGB";
+  const char* rgbaName = "RGBA";
+  if (mesh->GetPointData()->HasArray(rgbName))
   {
+    rgb = true;
+    arrayName = rgbName;
+  }
+  if (mesh->GetPointData()->HasArray(rgbaName))
```

Comment:
```suggestion
  else if (mesh->GetPointData()->HasArray(rgbaName))
```

Is it possible for a mesh to have both? I would think not, but you may or may not also want to check for RGBA first.

Reply:
The way its implemented, it checks for RGBA last so it uses that for coloring if a dataset has both RGB and RGBA. I think that is reasonable. We can refine this in a future PR and maybe give the user a choice.

Reply:
The way it's implemented is premature pessimization. (Note, I suggested also switching the checks for RGB and RGBA.) Always checking for RGB and then potentially discarding that work is silly.

Reply:
Sounds good.